lib/widgets/context_menu/context_menu_entry.dart

```diff
@@ -78,13 +81,10 @@ class _ContextMenuEntryState extends State<ContextMenuEntry> {
             if (widget.leading != null) ...[
               IconTheme.merge(
                 data: IconThemeData(
-                  size: 20,
+                  size: menuTheme.iconSize,
                   color: widget.enabled
-                      ? Theme.of(context).colorScheme.onSurface.withOpacity(0.7)
-                      : Theme.of(context)
-                          .colorScheme
-                          .onSurface
-                          .withOpacity(0.3),
+                      ? menuTheme.iconColor
+                      : menuTheme.disabledTextColor,
```

Comment:
menuTheme.disabledIconColor?

Reply:
Heh, just a copy-paste error. Already fix this mistake